server/sonar-server/src/main/java/org/sonar/server/es/request/ProxyCountRequestBuilder.java

```diff
@@ -31,7 +32,7 @@
 public class ProxyCountRequestBuilder extends CountRequestBuilder {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "CountRequestBuilder"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)